changelogs/fragments/611-gitlab-runners-env-vars-intput-and-default-item-limit.yaml

```
@@ -0,0 +1,2 @@
+minor_changes:
+  - gitlab_runners inventory plugin - permit environment variable input for ``server_url``, ``api_token`` and ``filter`` options (https://github.com/ansible-collections/community.general/pull/611).
```

```suggestion
  - gitlab_runners - permit environment variable input for ``server_url``, ``api_token`` and ``filter`` options (https://github.com/ansible-collections/community.general/pull/611).
```
I fixed that example of fragment a couple of days ago.

---

ok, there's no reason to wait